content/docs/forms.md

```diff
@@ -127,24 +127,24 @@ class EssayForm extends React.Component {
 
 ## सेलेक्ट टैग {#the-select-tag}
 
-HTML में, `<select>` एक ड्राप-डाउन लिस्ट बनाता है। उदाहरण के तौर पर, यह HTML फ्लेवर्स की एक ड्राप-डाउन लिस्ट बनाती है:
+HTML में, `<select>` एक ड्राप-डाउन लिस्ट बनाता है। उदाहरण के तौर पर, यह HTML, फ्लेवर्स की एक ड्राप-डाउन लिस्ट बनाती है:
 
 ```html
 <select>
-  <option value="grapefruit">Grapefruit</option>
-  <option value="lime">Lime</option>
-  <option selected value="coconut">Coconut</option>
-  <option value="mango">Mango</option>
+  <option value="चकोतरा">Grapefruit</option>
```

Comment:
The attribute field should be in English, else the code will not work. The value was supposed to be changed. So:

```  <option value="grapefruit">चकोतरा</option>``` and so on...